### PR TITLE
Fix vulnerabily warnings

### LINKED ISSUE #22: Update PostCSS
https://evilmartians.com/chronicles/postcss-8-is-coming-here-is-what-it-brings

### PR DESCRIPTION
**Pull request by trasherdk**
Fix vulnerabily warnings 
- bump deps to latest 
- migrate to @rollup/plugin-babel 
- add deprecated list

### CODE REVIEW

**Comment by trasherdk** [comment]
Cool. Tests failing because my Node is up to date :smiley: 

```sh
$ npm run build                                                                                                                  

> shortstack@1.1.7 build
> rimraf dist && concurrently npm:build:pre:* && concurrently npm:build:post:*

[build:pre:*css] 
[build:pre:*css] > shortstack@1.1.7 build:pre:css
[build:pre:*css] > postcss app/css/index.css -o dist/bundle.css --env production
[build:pre:*css] 
[build:pre:*js] 
[build:pre:*js] > shortstack@1.1.7 build:pre:js
[build:pre:*js] > rollup -c --environment NODE_ENV:production
[build:pre:*js] 
[build:pre:*js] 
[build:pre:*js] app/js/index.js → dist/bundle.js...
[build:pre:*js] babelHelpers: 'bundled' option was used by default. It is recommended to configure this option explicitly, read more here: https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers
[build:pre:*css] npm run build:pre:css exited with code 0
[build:pre:*js] created dist/bundle.js in 1.4s
[build:pre:*js] npm run build:pre:js exited with code 0
[build:post:*copy] 
[build:post:*copy] > shortstack@1.1.7 build:post:copy
[build:post:*copy] > cp app/index.html dist/index.html
[build:post:*copy] 
[build:post:*copy] npm run build:post:copy exited with code 0
```

File: deprecated.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Stuff need attention
```
Comment (argyleink) [inline deprecated.md]: can this become a github issue instead of a new file?

File: package.json [hunk 2]
```diff
@@ -21,25 +21,25 @@
...
+    "postcss-loader": "^6.2.1",
+    "postcss-preset-env": "7.2.3",
     "rimraf": "^3.0.2",
-    "rollup": "2.12.0",
-    "rollup-plugin-babel": "^4.4.0",
-    "rollup-plugin-postcss": "^3.1.1",
-    "rollup-plugin-terser": "^6.1.0"
+    "rollup": "2.66.1",
+    "rollup-plugin-postcss": "^4.0.2",
+    "rollup-plugin-terser": "^7.0.2"
```
Comment (argyleink) [inline package.json:43]: rad, thx for gettin all these, it'd been a minute 👍🏻

**Review by argyleink — APPROVED** [review]
this is great ty!
i asked if the deprecated notes could become an issue, but otherwise i have no change requests 🙏🏻

**Comment by argyleink** [comment]
fixes #22

File: deprecated.md (hunk 1 above)
Comment (trasherdk) [reply deprecated.md]: No problem. Deleted..

**Comment by trasherdk** [comment]
Damn, there is just no way to keep up :cry: 

Should I do a PR for the latest?
```
 @babel/core         ^7.16.12  →  ^7.17.0     
 cssnano              ^5.0.16  →  ^5.0.17     
 postcss                8.4.5  →    8.4.6     
 postcss-preset-env     7.2.3  →    7.3.1     
 rollup                2.66.1  →   2.67.1     
```

**Comment by argyleink** [comment]
lol sure!

**Pull request merged**